cypress/e2e/common/cmsSteps.js

```diff
@@ -8,8 +8,7 @@ const siteSection = Cypress.env('test_site_section');
 const randomStr = Cypress.env('randomStr');
 const frontEndBaseUrl = Cypress.env('front_end_base_url');
 
-When('user enters credentials', () => {
-    expect(username, 'username was set').to.be.a('string').and.not.be.empty
+When('user enters credentials of {string}', (username) => {
```

Comment:
What's going on here?  This is now using a userid that's passed in from the `.feature` file, but it's followed by a test for whether the **admin** password is present.

Reply:
After discussion with @KateMashkinaNIH, all test accounts use the same password.  This is about the variable name for passing it.